flexmeasures/ui/crud/assets.py

```diff
@@ -267,9 +284,17 @@ def get(self, id: str):
         asset = process_internal_api_response(asset_dict, int(id), make_obj=True)
         asset_form.process(data=process_internal_api_response(asset_dict))
 
+        account_id, account_name, parent_asset, account_url = self.get_breadcrumb_info(
+            asset
+        )
+
         return render_flexmeasures_template(
             "crud/asset.html",
             asset=asset,
+            account_id=account_id,
+            account_name=account_name,
+            account_url=account_url,
+            parent_asset=parent_asset,
```

Comment:
This code seems repeated in any call to `render_flexmeasures_template` that gets passed an `asset`. Therefore, it could move into the render method: if the method kwargs contain `asset`, then create breadcrumbs.

Reply:
Nice idea! I've moved the creation of the breadcrumb info to `render_flexmeasures_template`